setup.py

```diff
@@ -36,7 +36,7 @@ def find_version(*file_paths):
     license="APACHE2",
     author="SMART Platforms Team",
     author_email='support@smarthealthit.org',
-    packages=find_packages(exclude=['test*', '*_tests.py']),
+    packages=find_packages(exclude=['tests*']),
     install_requires=['requests', 'isodate'],
     classifiers=[
```

Comment:
Should the contents of this get updated?

Reply:
Of the classifiers? Almost certainly, yes. But I'd like to postpone that for a general "move to pyproject.toml" PR